bundle/src/main/java/com/jfrog/ide/eclipse/configuration/XrayServerConfigImpl.java

```diff
@@ -51,21 +53,58 @@ private String getValue(String key) {
 	}
 
 	@Override
-	public ProxyConfig getProxyConfForTargetUrl(String xrayUrl) {
+	public ProxyConfiguration getProxyConfForTargetUrl(String xrayUrl) {
 		xrayUrl = StringUtils.defaultIfBlank(xrayUrl, getUrl());
 		IProxyService service = ProxyManager.getProxyManager();
 		IProxyData[] proxyData = service.select(URI.create(xrayUrl));
 		if (ArrayUtils.isEmpty(proxyData)) {
 			return null;
 		}
 
-		ProxyConfig proxyConfig = new ProxyConfig();
-		proxyConfig.setHost(trim(proxyData[0].getHost()));
-		proxyConfig.setPort(proxyData[0].getPort());
+		ProxyConfiguration proxyConfig = new ProxyConfiguration();
+		proxyConfig.host = trim(proxyData[0].getHost());
+		proxyConfig.port = proxyData[0].getPort();
 		if (proxyData[0].isRequiresAuthentication()) {
-			proxyConfig.setUsername(trim(proxyData[0].getUserId()));
-			proxyConfig.setPassword(proxyData[0].getPassword());
+			proxyConfig.username = trim(proxyData[0].getUserId());
+			proxyConfig.password = proxyData[0].getPassword();
 		}
 		return proxyConfig;
 	}
+
+	@Override
```

Comment:
Why do we have the empty `@override`s?
Can we either delete (if possible), and if it's abstract - than shouldn't we implement?

Reply:
Implemented relevant methods.